docker-compose.yml

```diff
@@ -62,7 +62,7 @@ services:
       test: ps ax | grep elastic
 
   mysql.docker:
-    image: mysql
+    image: mysql:5.7
```

Comment:
[kinda optional] in a perfect world we should find a workaround for this, as this means that our lib doesn't guarantee to work with the most recent mysql version

Reply:
In this case we could update to `mysql:8.0` and the latest `pymysql` and it would work fine, but I guess we have to pick a side when a major update is released (because then with the same argument, our lib doesn't guarantee that it works with `mysql:5.7` for example). 

We already pin versions for the other dockers (C*, ES, Redis) by the way.